api/config.yaml

```diff
@@ -121,9 +121,9 @@ safety:
     git clone -b %GIT_BRANCH% --single-branch %GIT_REPO% code --quiet 2> /tmp/errorGitCloneSafety
     if [ $? -eq 0 ]; then
       cd code
-      cat *requirements* | grep '=' | grep -v '#' > safety_analysis_requirements.txt 2> /tmp/errorGetRequirements
```

Comment:
Just trying to avoid some misunderstood during CI: "My project does not contain this file!". Adding `huskyci` to it's name may mitigate this a little bit.